features/overlijden-indicatie-overleden.feature

```diff
@@ -0,0 +1,145 @@
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                               |
+      | type                | RaadpleegMetBurgerservicenummer      |
+      | burgerservicenummer | 000000152                            |
+      | fields              | burgerservicenummer,overlijden.datum |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000152 |
+      En heeft de persoon de volgende 'overlijden' gegevens
+      | naam               | waarde      |
+      | indicatieOverleden | true        |
+      | datum.type         | Datum       |
+      | datum.datum        | 2002-07-01  |
+      | datum.langFormaat  | 1 juli 2002 |
+
+    Abstract Scenario: overleden persoon wordt geraadpleegd met burgerservicenummer, <field> overlijden is onbekend en wordt gevraagd
+      Gegeven de persoon met burgerservicenummer '000000152' heeft de volgende 'overlijden' gegevens
+      | naam                     | waarde   |
+      | datum overlijden (08.10) | 20020701 |
+      | <naam gba element>       | 0000     |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                                 |
+      | type                | RaadpleegMetBurgerservicenummer        |
+      | burgerservicenummer | 000000152                              |
+      | fields              | burgerservicenummer,overlijden.<field> |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam                          | waarde    |
+      | burgerservicenummer           | 000000152 |
+      | overlijden.indicatieOverleden | true      |
+
+      Voorbeelden:
+      | naam gba element          | field  |
+      | plaats overlijden (08.20) | plaats |
+      | land overlijden (08.30)   | land   |
+
+  Rule: leeftijd veld wordt niet geleverd voor overleden personen
+
+    Scenario: overleden persoon wordt geraadpleegd met burgerservicenummer, leeftijd wordt gevraagd
+      Gegeven de persoon met burgerservicenummer '000000152' heeft de volgende 'overlijden' gegevens
+      | datum overlijden (08.10) | plaats overlijden (08.20) | land overlijden (08.30) |
+      | 20020701                 | 0518                      | 6030                    |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 000000152                       |
+      | fields              | leeftijd                        |
+      Dan heeft de response een persoon met alleen de volgende 'overlijden' gegevens
+      | naam               | waarde |
+      | indicatieOverleden | true   |
+
+  Rule: indicatieOverleden is in onderzoek wanneer ten minste één van de andere overlijden gegevens in onderzoek zijn
```

Comment:
geldt denk ik niet voor plaats en land, maar misschien wel voor Opschorting reden O?
Kan reden opschorting in onderzoek zijn?